db_json_util.py

```diff
@@ -59,62 +59,50 @@ def main() -> None:
-                    print(f'exported chains to {path_chains}')
-                elif path_chains.exists():
-                    user_input = input("File already exists for json chain, overwrite? (y/n): ")
-                    if user_input in ['y', 'Y', 'yes', 'Yes', 'YES']:
-                        with open(path_chains, 'w', encoding='utf-8') as f:
-                            json.dump(data_chains, f, indent=4)
-                        print(f'exported chains to {path_chains}')
-                    else:
-                        print('exiting, no data exported')
-                else:
-                    print('error occured, exiting')
-
-            if args.json_rpc_urls:
-                # get data and store it
-                response = requests.get(args.url + '/all/chains')
-                if response.status_code == 200:
-                    data_urls = response.json()
-                else:
-                    print(response.text)
-                    return
-                # check if path exists
-                if not path_urls.exists():
-                    os.makedirs(os.path.dirname(path_urls), exist_ok=True)
-                    with open(path_urls, 'w', encoding='utf-8') as f:
-                        json.dump(data_urls, f, indent=4)
-                    print(f'exported chains to {path_urls}')
-                elif path_urls.exists():
-                    user_input = input("File already exists for json rpc url, overwrite? (y/n): ")
-                    if user_input in ['y', 'Y', 'yes', 'Yes', 'YES']:
-                        with open(path_urls, 'w', encoding='utf-8') as f:
-                            json.dump(data_urls, f, indent=4)
-                        print(f'exported rpc urls to {path_urls}')
-                    else:
-                        print('exiting, no data exported')
-                    return
-                else:
-                    print('error occured, exiting')
+            api_export_to_json_files(path_chains, path_urls, args.url)
 
     if args.local:
         if args.import_data:
             local_import_from_json_files(path_chains, path_urls, path_db_file)
         if args.export_data:
             local_export_to_json_files(path_chains, path_urls, path_db_file)
 
+def export_to_file(file_name: Path, data: dict):
+    """
+    Writes data to a file.
+    """
+    with open(file_name, 'w', encoding='utf-8') as f:
+        json.dump(data, f, indent=4)
+        print(f'exported data to {file_name}')
+
+def api_export_json(path: Path, url: str):
+    if path.match(TABLE_CHAINS + '.json'):
+        response = requests.get(url + '/all/chains')
+    elif path.match(TABLE_RPC_URLS + '.json'):
+        response = requests.get(url + '/all/rpc_urls')
+    if response.status_code == 200:
+        data_chains = response.json()
+    else:
+        print(response.text)
+    # check if path exists
+    if not path.exists():
+        os.makedirs(os.path.dirname(path), exist_ok=True)
```

Comment:
Use the inbuilt (in `Path`) function `mkdir` instead of `os`:

```suggestion
        path.mkdir(parents=True, exist_ok=True)
```

With that I think we can remove `import os` as well, you could check if something else uses it and if not just remove it.